README.md

```diff
@@ -493,6 +493,12 @@ name,iterations,real_time,cpu_time,bytes_per_second,items_per_second,label
 "BM_SetInsert/1024/10",106365,17238.4,8421.53,4.74973e+06,1.18743e+06,
 ```
 
+## Output Files
+The library supports writing the output of the benchmark to a file specified
+by `--benchmark_output=<filename>`. The format of the output can be specified
+using `--benchmark_output_format={json|console|csv}`. Specifying
+`--benchmark_output` does not suppress the console output.
```

Comment:
There's a big difference between output for humans and output for tooling. Simply put I can't grok JSON and tooling can't grok colored console output, so I don't want to choose between the two. I want the both.

I agree the library shouldn't write to files in general, but `-o output` is a pretty universal feature.